src/commands/ban.ts

```diff
@@ -137,23 +133,24 @@ async function banHandler(interaction) {
 		}).catch(() => console.log('Failed to DM user'));
 
 		await member.ban({
-			reason
+			reason: reason
 		});
 
-		await Bans.create({
+		await Ban.create({
 			user_id: member.id,
-			admin_user_id: executingMember.id,
+			admin_user_id: executor.id,
 			reason: reason
 		});
 
-		bansListEmbed.addField(`${member.user.username}'s bans`, (count + 1).toString(), true);
+		bansListEmbed.addFields([
+			{ name: `${member.user.username}'s bans`, value: (count + 1).toString(), inline: true }
+		]);
 	}
 
-	await interaction.editReply({ embeds: [bansListEmbed], ephemeral: true });
+	await interaction.editReply({ embeds: [bansListEmbed] });
 }
 
 const command = new SlashCommandBuilder()
-	.setDefaultPermission(false)
```

Comment:
This is deprecated and does nothing in discordjs 14

Reply:
It was replaced with `setDefaultMemberPermissions`, and it doesn't mention what the default is without this. Since this is a moderation bot I would really like to ensure that by default no one has access to these commands

Reply:
I thought that not setting anything was the equivalent default, I've found [this](https://github.com/discordjs/discord.js/blob/main/packages/builders/src/interactions/slashCommands/mixins/SharedSlashCommand.ts#L94-L95) which shows that we should use
```ts
command.setDefaultMemberPermissions('0');
```
It looks weird but that's what it recommends to disable it by default